webpack.config.js

```diff
@@ -206,6 +206,18 @@ const config = {
         to: "favicon.ico"
       }
     ]),
+    new CopyWebpackPlugin([
```

Comment:
We'll need to whitelist these in reticulum here so they will be served statically from origin: https://github.com/mozilla/reticulum/blob/master/lib/ret_web/endpoint.ex#L16

If we want these CDN'ed, we'll need to digest them and presumably add some absolute URLs somewhere to point the browser to them (I'm too unfamililar with the spec to know how to do this offhand but I'm guessing it's possible?)

Reply:
the URLs for any `src` value in the manifest (e.g., for `icons` and `screenshots`) can be local or external. local URLs are relative to the manifest URL (just FYI).

I'll follow up with this. I think I'll try [this small `webpack-pwa-manifest` Webpack plugin](https://github.com/arthurbergmz/webpack-pwa-manifest#webpack-pwa-manifest), and see how that works out. thanks for the helpful comments!